src/rabbit_misc.erl

```diff
@@ -263,6 +264,7 @@
 -spec get_gc_info(pid()) -> [any()].
 -spec group_proplists_by(fun((proplists:proplist()) -> any()),
                          list(proplists:proplist())) -> list(list(proplists:proplist())).
+-spec get_erl_path() -> {ok, string()}.
```

Comment:
For newer code, please put the spec near the function, it's far easier to read and keep up-to-date.

Reply:
Also, the correct type, as returned by `filename:join/2`, is `file:filename_all()`.